packages/spindle-ui/src/HeroCarousel/HeroCarousel.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { FC } from 'react';
+import PropTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+import ChevronLeftBold from '../Icon/ChevronLeftBold';
+import ChevronRightBold from '../Icon/ChevronRightBold';
+import Pause from '../Icon/Pause';
+import PlayFill from '../Icon/PlayFill';
+import type { CarouselItem } from './HeroCarouselItem';
+import HeroCarouselItem from './HeroCarouselItem';
+import { useAutoPlayCarousel } from './hooks/useAutoPlayCarousel';
+
+type Props = {
+  className?: string;
+  carouselList: CarouselItem[];
+};
+
+const ITEM_LINK_CLASS_NAME = 'js-auto-play-carousel-item-link';
+
+export const HeroCarousel: FC<Props> = React.memo(function AutoPlayCarousel({
```

Comment:
```suggestion
export const HeroCarousel: FC<Props> = React.memo(function HeroCarousel({
```
[nits] 細かいのですがここの命名も揃えられると良さげです！

Reply:
揃えました！ -> https://github.com/openameba/spindle/pull/348/commits/4c4abf09ad581df71a9db611812fb1b39aef21f4